ehrql/backends/tpp.py

```diff
@@ -218,10 +218,18 @@ def medications(self):
         """
             SELECT DISTINCT
                 Appointment_ID AS appointment_id,
+                CAST (BookedDate AS date) AS booked_date,
+                CAST(StartDate AS date) AS start_date,
```

Comment:
By moving these lines, we've introduced noise into the diff. We don't need to move them.